src/iso8583/server.rs

```diff
@@ -108,6 +108,11 @@ fn new_client(iso_server: &ISOServer, stream_: TcpStream) {
         'done:
         loop {
             if reading_mli {
+                match server.mli.is_available(&stream) {
+                    Ok(true) => true,
+                    IsoError => break 'done,
```

Comment:
shouldn't this be Err(e)=> break 'done,?

Reply:
Good catch, yes